svc/ClientHealthService.ts

```diff
@@ -144,31 +115,27 @@ export class ClientHealthService extends HoistService {
     // Implementation
     //------------------
     private sendReport() {
-        const {
-            intervalMins,
-            severity: defaultSeverity,
-            ...rest
-        } = XH.trackService.conf.clientHealthReport ?? {};
-
-        const rpt = this.getReport();
-        let severity = defaultSeverity ?? 'INFO';
-        if (find(rpt, (v: any) => isPlainObject(v) && v.severity === 'WARN')) {
-            severity = 'WARN';
-        }
+        const {intervalMins, ...rest} = XH.trackService.conf.clientHealthReport ?? {};
 
         XH.track({
             category: 'App',
             message: 'Submitted health report',
-            severity,
             ...rest,
-            data: rpt
+            data: {
+                clientId: XH.clientId,
+                sessionId: XH.sessionId,
+                report: this.getReport()
+            }
         });
     }
 }
 
-export interface SessionData {
+export interface GeneralData {
```

Comment:
`SessionData`?

Reply:
right -- was ceding the term "session" to the "sessionStorage" scope meaning, especially since we now have a 'sessionId'